modules/cluster-logging-feature-reference.adoc

```diff
@@ -0,0 +1,107 @@
+// Module is included in the following assemblies:
+//cluster-logging-loki.adoc
+:_content-type: REFERENCE
+[id="logging-feature-ref_{context}"]
+= Collector features
+
+.Log Sources
+[options="header"]
+|===============================================================
+| Feature                   | Fluentd  | Vector
+| App container logs        | &#10003; | &#10003;
+| App specific routing      | &#10003; | &#10003;
+|...by namespace            | &#10003; | &#10003;
```

Comment:
Ellipses needed?

Reply:
To indicate app specific routing by (particular characteristic) as in namespace, if there's a different / better way to do that I'm certainly open to it.